2003/mtjin/AndroidArchitectureStudy/app/build.gradle

```diff
@@ -48,5 +50,13 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxjava:2.2.12"
     implementation "io.reactivex.rxjava2:rxandroid:2.1.1"
     implementation 'com.squareup.retrofit2:adapter-rxjava2:2.5.0'
+    // Room
+    implementation "androidx.room:room-runtime:2.2.4"
+    testImplementation "androidx.room:room-testing:2.2.4"
+    kapt "androidx.room:room-compiler:2.2.4"
+    annotationProcessor "androidx.room:room-compiler:2.2.4"
```

Comment:
못보고 넘어갈뻔 했는데 kapt와 annotationProcessor를 같이 사용하셨네요.
```kotlin
annotationProcessor "androidx.room:room-compiler:$room_version" 
// For Kotlin use kapt instead of annotationProcessor
```
한개만 선언해주세요.

Reply:
35f0211
0226020
수정했습니다.